doc/build-unix.md

```diff
@@ -137,8 +137,8 @@ turned off by default.  See the configure options for upnp behavior desired:
 
 To build:
 
-	tar -xzvf miniupnpc-1.6.tar.gz
-	cd miniupnpc-1.6
+	tar -xzvf miniupnpc-x.x.tar.gz
+	cd miniupnpc-x.x
 	make
 	sudo su
 	make install
```

Comment:
Hmm, thinking about it, I don't like recommending installing it globally at all
E.g. the berkeleydb build instructions describe how to install to a local dir and use it there